Add `REQUIRE_PARTS` setting (#703)
scrapinghub/dateparser: docs/usage.rst

changes the parsers to try without skipping parsers that may be added to
Dateparser in the future. For example, to ignore relative times:

    >>> from dateparser.settings import default_parsers
    >>> parsers = [parser for parser in default_parsers if parser != 'relative-time']
    >>> parse('today', settings={'PARSERS': parsers})

> ``REQUIRE_PARTS`` This option ensures results are dates that have some specified part. It defaults to ``[]`` and can include ``day``, ``month`` and/or ``year``.

Reviewer: ```suggestion
``REQUIRE_PARTS`` This option ensures results are dates that have all specified parts. It defaults to ``[]`` and can include ``day``, ``month`` and/or ``year``.
```
